crates/astria-composer/src/searcher/collector.rs

```diff
@@ -44,7 +45,8 @@ pub(super) struct Transaction {
 pub(super) struct Collector {
     // Chain ID to identify in the astria sequencer block which rollup a serialized sequencer
     // action belongs to.
-    chain_id: String,
+    chain_id: ChainId,
+    chain_name: String,
```

Comment:
why do we need to store this?

Reply:
Mainly to give `Collector` an identity. This will be nice once we have it report its state (in a health check) and when constructing its scope when calling its `run` function (or whatever it's called).